Engine/EditorUI/Helper/ImGuiHelper.h

```diff
@@ -0,0 +1,184 @@
+#ifndef _IMGUIHELPER_H_
```

Comment:
maybe here we could make a template also for GO substituting the DropGOTarget that is now used

Reply:
Yeah, right!